### ISSUE TITLE
Bitrix Push server 2.0

### ISSUE THREAD

**Issue by Forsakenrox** [bitrixdock/bitrixdock#225]
Есть идеи того, как это можно реализовать в рамках данного образа ? Чаты и видеозвонки не работают, а поиск по интернету не даёт ничего вразумительного 😥

**Comment by maxisoft-git** [comment]
На текущий момент это не реализуемо так как нужно подымать NodeJS для реализации push сервера.

**Comment by Forsakenrox** [comment]
> На текущий момент это не реализуемо так как нужно подымать NodeJS для реализации push сервера.

Да, но ведь у нас тут docker и вроде бы идея должна быть в том что бы поднять ещё 1 контейнер, или я не прав ?

**Comment by FaritSlv** [comment]
@Forsakenrox все верно. Вообще не понимаю в чем сложность реализовать? Вот пример как поднять и [откуда что скачать](https://dev.1c-bitrix.ru/learning/course/index.php?COURSE_ID=135&LESSON_ID=2376), создайте еще один докер файл, в него кладете все эти настройки и в nginx правильно указать имя контейнера куда ему отправлять запросы и все

**Comment by paskal** [comment]
Я посмотрю, это должно быть несложно реализовать.

**Comment by paskal** [comment]
Я был неправ, там всё нетривиально, и просто так не ложится на парадигму контейнеров. Попробую ещё раз когда дойдут руки.

**Comment by churneya** [comment]
https://github.com/bitrixdock/bitrixdock/pull/241

**Comment by Elnadrion** [comment]
Залили https://github.com/bitrixdock/bitrixdock/pull/241
